.github/CONTRIBUTING.md

```diff
@@ -121,22 +121,19 @@ Design tokens source files are in the `./tokens` directory, in YAML format.
 Adding, removing, and updating design tokens should be discussed in the issues of this project before contributing to code changes.
 
 Note: `./tokens/colors.yml` should not be manually edited, as it is automatically generated by this project.
-Colors are managed in Invision’s [Design System Manager](https://www.invisionapp.com/blog/announcing-invision-design-system-manager/) (see below).
+Colors are managed in Invision [Design System Manager](https://www.invisionapp.com/blog/announcing-invision-design-system-manager/) (see below).
 
 ---
 
 ## Updating color tokens
```

Comment:
I wonder if we should make info about consuming Polaris colors through DSM and Craft more visible in the style guide. Perhaps on the Resources page and/or the Polaris UI Kit page.